draft-ietf-taps-interface.md

```diff
@@ -756,8 +756,8 @@ Name:
 
 This property specifies whether the application needs to use a transport
 protocol that ensures that all data is received on the other side without
-corruption. This also entails being notified when a Connection is closed or
-aborted. The default is to Require Reliable Data Transfer.
+corruption. The default is to Require Reliable Data Transfer. This also
+entails being notified when a Connection is closed or aborted. 
```

Comment:
I don't see why switching the order here would be an improvement? Since the "entails" statement gives more detail about what the property's behavior, I think this reads better in the previous version. You must have had a reason to propose this... what am I missing?

Reply:
Maybe you are right. The point was that you only get this notification if this property is enabled. Maybe that's what we should say...

Reply:
Agreed - suggestion: switch the order back, but change the sentence - e.g.: "This also entails being notified when a Connection is closed or aborted when Reliable Data Transfer is enabled. The default is to Require Reliable Data Transfer."